### PR TITLE
[New] Match viewpoint of geo views

### PR DESCRIPTION
**Pull request by CalebRas**
## Description

This PR implements `Match viewpoint of geo views` in `Scenes` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-MatchViewpointOfGeoviews/Shared/Samples/Match%20viewpoint%20of%20geo%20views)

## Linked Issue(s)

- `swift/issues/5282`

## How To Test

- Pan and zoom on the map view or scene view to see the other view's viewpoint match the one interacted with.

## Screenshots

match-viewpoint-of-geo-views

### CODE REVIEW

File: Shared/Samples/Match viewpoint of geo views/MatchViewpointOfGeoViewsView.swift [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    /// A scene with an imagery basemap.
+    @State private var scene = Scene(basemapStyle: .arcGISImagery)
+    
+    /// The viewpoint of the map view.
+    @State private var mapViewpoint: Viewpoint?
+    
+    /// The viewpoint of the scene view.
+    @State private var sceneViewpoint: Viewpoint?
+    
+    /// A Boolean value indicating whether the map view is currently being navigated.
```
Comment (pgruenler) [inline Shared/Samples/Match viewpoint of geo views/MatchViewpointOfGeoViewsView.swift:31]:
```suggestion
    /// A Boolean value indicating whether the map view is currently navigating.
```
"being navigated" sounds like interaction, I think "navigating" is a better fit.

File: Shared/Samples/Match viewpoint of geo views/MatchViewpointOfGeoViewsView.swift [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    @State private var mapIsNavigating = false
+    
+    /// A Boolean value indicating whether the scene view is currently being navigated.
```
Comment (pgruenler) [inline Shared/Samples/Match viewpoint of geo views/MatchViewpointOfGeoViewsView.swift:34]:
```suggestion
    /// A Boolean value indicating whether the scene view is currently navigating.
```

**Review by pgruenler — COMMENTED** [review]
I left a couple suggestions.

**Review by njarecha — CHANGES_REQUESTED** [review]

**Review by njarecha — APPROVED** [review]

**Review by pgruenler — APPROVED** [review]

**Comment by CalebRas** [comment]
Thank you guys!

**Pull request merged**